picard/metadata.py

```diff
@@ -30,6 +30,21 @@
 
 MULTI_VALUED_JOINER = '; '
 
+# lengths difference over this number of milliseconds will give a score of 0.0
+# equal lengths will give a score of 1.0
+# example
+# a     b     score
+# 20000 0     0.333333333333
+# 20000 10000 0.666666666667
+# 20000 20000 1.0
+# 20000 30000 0.666666666667
+# 20000 40000 0.333333333333
+# 20000 50000 0.0
+LENGTH_SCORE_THRES_MS = 30000
```

Comment:
We can move this to picard.constants?

Reply:
I don't think it is needed, this value is very unlikely to be used elsewhere. If it happens in the future, we'll move it to picard.constants.

Reply:
Maybe I'm stupid, but none of these these calculations appear to be correct (and I'm not sure how they should be with the current implementation of `length_score`):

```
In [3]:     def length_score(a, b):
   ...:         return (0.0 - min(abs(a - b), LENGTH_SCORE_THRES_MS) /
   ...:                 float(LENGTH_SCORE_THRES_MS))
   ...:     
In [7]: for a,b in [(20000, 0), (20000, 10000), (20000, 20000), (20000, 30000), (20000, 40000), (20000, 50000)]:
   ...:     print(f"{a}, {b} = {length_score(a,b)}")
   ...:     
20000, 0 = -0.6666666666666666
20000, 10000 = -0.3333333333333333
20000, 20000 = 0.0
20000, 30000 = -0.3333333333333333
20000, 40000 = -0.6666666666666666
20000, 50000 = -1.0
```

In fact, if the score needs to be between 0.0 and 1.0, how is substracting an always positive number from 0.0 going to achieve that (no, changing `-` to `+` doesn't get you the numbers from the docstring).

Reply:
I think this happened sometime later while rebasing. It was 1.0 when I reviewed it.